File: 2025/java/senior/Senior1.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

public class Senior1 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer tokens = new StringTokenizer(reader.readLine());
        int a = Integer.parseInt(tokens.nextToken());
        int b = Integer.parseInt(tokens.nextToken());
        int x = Integer.parseInt(tokens.nextToken());
        int y = Integer.parseInt(tokens.nextToken());

        long area1 = (a + x) + Math.max(b, y);
        long area2 = (b + y) + Math.max(a, x);

        System.out.println(2 * Math.min(area1, area2));
    }
}

File: 2025/cpp/senior/senior1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int a, b, x, y;
  cin >> a >> b >> x >> y;

  int area1 = (a + x) + max(b, y);
  int area2 = (b + y) + max(a, x);

  cout << 2 * min(area1, area2);
}
